.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
     working_directory: /mnt/crate
     steps:
       - checkout
+      - run: git submodule update --init --remote
```

Comment:
See also the [recommended way to do it](https://discuss.circleci.com/t/can-not-get-submodules-to-clone-properly/22120) accordingly to CircleCI support staff.

Reply:
Or a slightly different [recipe](https://circleci.com/docs/2.0/configuration-reference/#checkout) in the reference doc.